bin/push_docker.js

```diff
@@ -51,20 +49,23 @@ const dependencies = {
 
 const configVersion = dependencies[configurationEntry].split('#')[1];
 const dockerRegistryDomain = 'quay.io';
-const repository = `${dockerRegistryDomain}/wire/webapp${suffix}`;
+const repository = `${dockerRegistryDomain}/wire/webapp${distributionParam ? `-${distributionParam}` : ''}`;
 
-const dockerImageTag = `${repository}:${pkg.version}-${commitShortSha}-${configVersion}${stage}`;
-const dockerImageStageTag = stageParam ? `${repository}:${stageParam}` : '';
+const tags = [];
+stageParam && tags.push(`${repository}:${stageParam}`);
+releaseParam && tags.push(`${repository}:${releaseParam}-${configVersion}-${commitShortSha}`);
```

Comment:
As discussed internally `pkg.version` is incremented by now. We are slowly transitioning to semver instead of a date based release version. But for now we can do whatever works best for you.

Reply:
Agreed. As soon as the release name (or part of it) is reflected by `pkg.version`, we are going to adjust the image tag name, which is supposed to contain the release name.